packages/client/mutations/AddReactjiToReactableMutation.ts

```diff
@@ -104,7 +113,19 @@ const AddReactjiToReactableMutation: StandardMutation<TAddReactjiToReactableMuta
         }
       }
     },
-    onCompleted,
+    onCompleted: (res, errors) => {
+      const {isRemove} = variables
+      const addedKudos = res.addReactjiToReactable.addedKudos
+      if (!isRemove && addedKudos) {
+        const {unicode} = getReactji(addedKudos.emoji)
+        atmosphere.eventEmitter.emit('addSnackbar', {
```

Comment:
+1 We don't seem to be emitting the "Snackbar Viewed" and "Snackbar Clicked" events for the kudos snackbar.  I don't think these are critical for our use case (since we're capturing kudos with the "Kudos Sent" event), but it would be nice to emit these events for every snackbar to be consistent.